### PR TITLE
#11 As a user I want separate views for users and teams. I also want them presented in a nice way.

### PR DESCRIPTION
**Pull request by xXEKEXx**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
+        <module name="OwlsOOPProject" />
```
Comment (PearPressure) [inline .idea/compiler.xml]: Vet inte varför du lägger till detta.

File: src/main/java/storagesystem/model/StoreIT.java [hunk 2]
```diff
@@ -1,109 +1,109 @@
-package storagesystem.model;
```
Comment (PearPressure) [inline src/main/java/storagesystem/model/StoreIT.java]: Du tar bort hela filen och lägger tillbaka den Svårt att se dina ändringar i filen.

File: src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+         <children>
+            <Label text="Team:" textFill="WHITE" AnchorPane.leftAnchor="14.0" AnchorPane.topAnchor="0.0">
+               <font>
+                  <Font size="61.0" />
+               </font>
+            </Label>
+            <ChoiceBox fx:id="settingsChooseTeamInput" layoutX="764.0" layoutY="291.0" prefWidth="150.0" />
+            <Text fill="WHITE" layoutX="757.0" layoutY="265.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Choose a team:" underline="true" />
+            <Button fx:id="editTeamButton" layoutX="866.0" layoutY="574.0" mnemonicParsing="false" onAction="#editTeamButtonPressed" prefHeight="70.0" prefWidth="126.0" styleClass="inAppButton" text="Edit team" AnchorPane.bottomAnchor="50.0" AnchorPane.rightAnchor="50.0" />
+            <TextArea fx:id="teamToCTextArea" editable="false" layoutX="332.0" layoutY="240.0" prefHeight="325.0" prefWidth="337.0" wrapText="true" />
```
Comment (Sponken) [inline src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml]: Feels like this should look better. Make the background same as the anchorpane and change text color to white

File: src/main/java/storagesystem/viewcontroller/settings/userPage.fxml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            <Text fx:id="contactInputEmptyError" fill="RED" layoutX="587.0" layoutY="258.0" opacity="0.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Fill in contact information!">
+               <font>
+                  <Font size="18.0" />
+               </font>
+            </Text>
+         </children>
+      </AnchorPane>
+      <AnchorPane fx:id="viewProfileAnchorPane" prefHeight="700.0" prefWidth="1000.0" styleClass="background">
+         <children>
+            <TextArea fx:id="profileDescriptionTextArea" editable="false" layoutX="340.0" layoutY="295.0" prefHeight="250.0" prefWidth="400.0" wrapText="true" AnchorPane.leftAnchor="340.0" AnchorPane.topAnchor="335.0">
```
Comment (Sponken) [inline src/main/java/storagesystem/viewcontroller/settings/userPage.fxml]: Should look the same as textfield

File: src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml [hunk 5]
```diff
@@ -40,6 +40,31 @@
             <Text fill="WHITE" layoutX="64.0" layoutY="203.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Team Name:" underline="true" wrappingWidth="150.00000596046448" AnchorPane.topAnchor="180.0" />
```
Comment (Sponken) [inline src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml]: Mouse changes on hover, which indicates you can edit it which you can't. Mouse shouldn't change

File: src/main/java/storagesystem/viewcontroller/FrameworkController.java [hunk 6]
```diff
@@ -54,7 +50,16 @@ public void initialize(URL location, ResourceBundle resources) {
...
 
     @FXML
-    void teamButtonPressed(){ loadUI("settings/teamPage"); }
+    void teamButtonPressed(){
+        if(StoreIT.getCurrentOrganisation().getUsersTeams(StoreIT.getCurrentUser()).size() > 0){
+            loadUI("settings/teamPage");
+        } else{
+            NoTeamPopUpController popUp = new NoTeamPopUpController();
+            rootPane.getChildren().add(popUp);
+            AbstractFader.fadeTransition(popUp, 3);
```
Comment (steget) [inline src/main/java/storagesystem/viewcontroller/FrameworkController.java]: I'd say the animation is very quick now, barely enough time to read it, change to ~6-7sec

File: src/main/java/storagesystem/viewcontroller/settings/userPage.fxml [hunk 7]
```diff
@@ -53,6 +49,7 @@
                   <Font size="18.0" />
                </font>
             </Text>
+            <TextArea fx:id="profileDescriptionInput" layoutX="340.0" layoutY="335.0" prefHeight="250.0" prefWidth="400.0" />
```
Comment (Sponken) [inline src/main/java/storagesystem/viewcontroller/settings/userPage.fxml]: Mouse shouldn't change when you can't edit

File: src/main/java/storagesystem/viewcontroller/settings/TeamPageController.java [hunk 8]
```diff
@@ -130,8 +140,8 @@ private void fillTeamAttributes() {
      */
     @FXML
     public void saveTeam() {
-        currentlySelectedTeam.setTermsAndConditions(settingsTeamContractInput.getText());
-        currentlySelectedTeam.setName(settingsTeamNameInput.getText());
+        StoreIT.getCurrentTeam().setTermsAndConditions(settingsTeamContractInput.getText());
+        StoreIT.getCurrentTeam().setName(settingsTeamNameInput.getText());
```
Comment (steget) [inline src/main/java/storagesystem/viewcontroller/settings/TeamPageController.java]: Gör man sitt lagnamn super långt så kan man inte byta lag senare för då öppnas inte choiceboxen upp.
Borde kontrolleras här

File: src/main/java/storagesystem/viewcontroller/settings/TeamPageController.java [hunk 9]
```diff
@@ -204,17 +214,17 @@ private void removeMemberButtonPressed() {
     private void removeMemberFromTeam(int userID) {
         boolean memberFound = false;
         //remove member from team
-        for (int i : currentlySelectedTeam.getAllMemberIDs()) {
+        for (int i : StoreIT.getCurrentTeam().getAllMemberIDs()) {
```
Comment (steget) [inline src/main/java/storagesystem/viewcontroller/settings/TeamPageController.java]: can be  changed to
`if (StoreIT.getCurrentTeam().getAllMemberIDs().contains(userID)) {`

**Pull request merged**